CONTRIBUTING.md

```diff
@@ -1,55 +1,77 @@
+### How Do I Submit A (Good) Bug Report?
 
-We love contributions from our community! If you'd like to go a step beyond contributor
-and become a committer with full write access and a say in the project, you must
-be invited to the project. The existing committers employ an internal nomination
-process that must reach lazy consensus (silence is approval) before invitations
-are issued. If you feel you are qualified and want to get more deeply involved,
-feel free to reach out to existing committers to have a conversation about that.
+Bugs are tracked as [GitHub issues](https://guides.github.com/features/issues/). Create an issue on the repository and provide the following information by filling in [the template](https://github.com/adobe/spectrum-web-components/blob/main/.github/ISSUE_TEMPLATE/bug_report.yaml).
 
-## Security Issues
+[IMPORTANT] Explain the problem and include additional details to help maintainers reproduce the problem:
+
+-   **Describe the exact steps which reproduce the problem** in as many details as possible.
+-   **Provide specific examples to demonstrate the steps**. If you're providing snippets in the issue, use [Playground](https://studio.webcomponents.dev/workspace/adobe).
+-   **Explain which behavior you expected to see instead and why.**
+-   **Include screenshots and animated GIFs** which show you following the described steps and clearly demonstrate the problem.
+
+## Do you intend to add a new feature or change an existing one?
+
+-   Suggest your change in the swc-core mailing list and start writing code.
+
+-   Do not open an issue on GitHub until you have collected positive feedback about the change. You can start on [Adding your idea here](https://github.com/adobe/spectrum-web-components/discussions/categories/ideas) under Ideas
+
+-   Once your Idea or request is submitted you can start creating a [issue](https://github.com/adobe/spectrum-web-components/issues/new/choose) with New Component or a Feature Request
+
+-   Please make sure that your PR follows our best practices and styleguide before you submit for review
+
+## Create your first Pull Request
+
+Please follow these steps to have your contribution considered by the maintainers:
+
+1. Follow all instructions in [the template](PULL_REQUEST_TEMPLATE.md)
+2. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) are passing.
+
+## Git Commit Messages
+
+We maintain a specific standard of commit messages while committing to the respository. See [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
+
+## Do you want to contribute to the Spectrum Web Components documentation?
+
+-   Please read [Contributing Guidelines for documentation](https://github.com/adobe/spectrum-web-components#documentation)
+
+## Pull Request Labels
+
+Specific to your contribution and the status we would encourage you to use PR labels such as `WIP`, `triage` , `under-review`, `testing`
```

Comment:
I think other. than "WIP", labels aren't necessary from my perspective. As long as the PR is linked to an issue, that helps, and you can flag a PR as a draft. I think it's good to specify reviewers and notify them when your PR is ready for review. Let me know what you think.

Reply:
Yea make sense. A little bit of overhead i think. Lets not add this specification. I think it would be self explanatory.